ttnn/ttnn/model_preprocessing.py

```diff
@@ -39,6 +42,45 @@ def preprocess_embedding_weight(weight, *, dtype):
     return weight
 
 
+def preprocess_conv2d(weight, bias, ttnn_module_args):
+    if ttnn_module_args is None:
+        raise RuntimeError(f"torch.nn.Conv2d modules need run_model to be provided to preprocess_model_parameters")
+
+    weight = ttnn.from_torch(weight, dtype=ttnn.bfloat16)
+    if bias is not None:
+        bias = ttnn.from_torch(torch.reshape(bias, (1, 1, 1, -1)), dtype=ttnn.bfloat16)
+
+    conv = ttnn.Conv2D(
+        **ttnn_module_args,
+        weight=weight,
+        bias=bias,
+        reader_patterns_cache=None,
+        conv_blocking_and_parallelization_config_override=None,
+        move_weights_to_device=False,
+        device=None,
+    )
+
+    parameters = {}
+    parameters["weight"] = ttnn.Tensor(conv.conv.weight)
+    if bias is not None:
+        parameters["bias"] = ttnn.Tensor(conv.conv.bias)
+    return parameters
+
+
+def fold_batch_norm2d_into_conv2d(conv, bn):
+    weight = conv.weight
+    bias = conv.bias
+    running_mean = bn.running_mean
+    running_var = bn.running_var
+    eps = bn.eps
+    scale = bn.weight
+    shift = bn.bias
+    weight = weight * (scale / torch.sqrt(running_var + eps))[:, None, None, None]
+    if bias is not None:
+        bias = (bias - running_mean) * (scale / torch.sqrt(running_var + eps)) + shift
```

Comment:
Aren't `running_mean` and `shift` unaccounted for if conv bias isn't present?  This might never happen in real models though